src/main/java/org/reaktivity/specification/socks/internal/SocksFunctions.java

```diff
@@ -153,13 +156,27 @@ public String getPrefixName()
     public static Consumer<Builder> addressBuilder(
```

Comment:
will this method generate garbage?

Reply:
ideally no, but not sure, i need to review the JVM behavior for this case